lib/extreme.ex

```diff
@@ -95,6 +106,16 @@ defmodule Commanded.EventStore.Adapters.Extreme do
     end
   end
 
+  @spec subscribe_to_all_streams(String.t(), pid, Commanded.EventStore.start_from()) ::
+          {:ok, subscription :: pid}
+          | {:error, :subscription_already_exists}
+          | {:error, term}
+  def subscribe_to_all_streams(subscription_name, subscriber, start_from \\ :origin)
+
+  def subscribe_to_all_streams(subscription_name, subscriber, start_from) do
+    subscribe_to_stream("", subscription_name, subscriber, start_from)
```

Comment:
I suggest you use `$all` here instead. Not sure what the `""` will actually accomplish...